### PR TITLE
CDH-XXX: Minor UI enhancements

### PR DESCRIPTION
**Pull request by cheinz-sag**
Hi Tim and Marlene,

can you pls review?

@MWindel I'd propose to wait for Tim's feedback before you start reviewing as he'll check also for correctness of the content.

The PR is to be merged to:
- develop
- release/y2024

Thanks
Christoph

### CODE REVIEW

File: content/datahub/working-with-datahub-bundle/overview-offloading.md [hunk 1]
```diff
@@ -58,15 +58,17 @@ The target table name refers to the target table in Dremio, with which the data
...
+The offloading status is empty if the offloading has not been executed yet. For running and completed executions the start time is shown and either a calendar icon for a scheduled execution or a user icon for a manually triggered execution. For a running execution the elapsed time is additionally shown. For a completed execution the failure/success status, the number of offloaded records and the runtime are shown as well. For running and completed executions clicking on the offloading status navigates you to the detail view for that execution in the job history.
```
Comment (MWindel) [inline content/datahub/working-with-datahub-bundle/overview-offloading.md:61]:
```suggestion
The offloading status is empty if the offloading has not been executed yet. For running and completed executions the start time is shown and either a calendar icon for a scheduled execution or a user icon for a manually triggered execution. For a running execution the elapsed time is additionally shown. For a completed execution the failure/success status, the number of offloaded records and the runtime are shown as well. For running and completed executions click the offloading status to navigate to the detail view for that execution in the job history.
```

File: content/datahub/working-with-datahub-bundle/overview-offloading.md [hunk 2]
```diff
@@ -58,15 +58,17 @@ The target table name refers to the target table in Dremio, with which the data
...
+
+The sort control allows for sorting by successful/failed jobs. The filter control allows for filtering by execution status.
```
Comment (MWindel) [inline content/datahub/working-with-datahub-bundle/overview-offloading.md:63]:
```suggestion
The sort control allows you to sort by successful/failed jobs. The filter control allows you to filter by execution status.
```

File: content/datahub/working-with-datahub-bundle/monitoring-compaction-jobs.md [hunk 3]
```diff
@@ -46,27 +46,28 @@ The list shows the execution history with each execution consisting of the follo
...
 Click **Reload** to refresh the list.
 
-You can filter the entries by their status or timestamp by using the filter controls at the top.
+You can filter the entries by their status or timestamp by using the filter controls at the top. Click **Apply** to filter entries with the current filter settings. Click **Reset filter** to reset the current filter settings. Per default the entries from the last seven days are shown.
 
 ### Details of compaction job {#details-of-compaction-job}
 
 For a given compaction job, you can examine additional details of its execution.
 
 In the corresponding list of jobs click on the job you are specifically interested in. A details view encompasses the following information:
```
Comment (MWindel) [inline content/datahub/working-with-datahub-bundle/monitoring-compaction-jobs.md:62]:
```suggestion
In the corresponding list of jobs click the job you are specifically interested in. A details view encompasses the following information:
```

File: content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md [hunk 4]
```diff
@@ -49,13 +49,14 @@ For a given offloading job, you can examine additional details of its execution.
 
 In the corresponding list of jobs click on the job you are specifically interested in. A details view encompasses the following information:
```
Comment (MWindel) [inline content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md:50]:
```suggestion
In the corresponding list of jobs click the job you are specifically interested in. A details view encompasses the following information:
```

File: content/datahub/working-with-datahub-bundle/overview-offloading.md (hunk 1 above)
Comment (doernemt) [inline content/datahub/working-with-datahub-bundle/overview-offloading.md:61]: The offloading status is empty if the offloading has not been executed yet. For running and completed executions, the start time is shown and either a calendar icon for a scheduled execution or a user icon for a manually triggered execution. For a running execution, the elapsed time is additionally shown. For a completed execution, the failure/success status, the number of offloaded records and the runtime are shown as well. For running and completed executions, clicking on the offloading status navigates you to the detail view for that execution in the job history.

**Review by doernemt — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**